src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/UpdateFlowAction.java

```diff
@@ -234,10 +234,10 @@ private String getOrCreateFlowGroupId(String flowId) throws FlowProcessingExcept
     }
 
     private FlowLoopOperation detectFlowLoopOperation(RequestedFlow originalFlow, RequestedFlow targetFlow) {
-        if (originalFlow.getLoopSwitchId() == targetFlow.getLoopSwitchId()) {
+        if (originalFlow.getLoopSwitchId() == null && targetFlow.getLoopSwitchId() == null) {
```

Comment:
what if `originalFlow.getLoopSwitchId() == targetFlow.getLoopSwitchId()` 
will we create same second rule in this case? Why we shouldn't return `NONE`?

Reply:
Requirements here was to reinstall loop rules when we have second create flow loop request with the same params. To achieve this goal when we detect request to create flow loop for already looped flow with the same switchId we need to process `CREATE` operation.  `FlowLoopOperation.NONE` is used for endpoint update only (when changed port/vlan) and we need to install new rules and delete old ones.